cmd/dcrdata/public/scss/utils.scss

```diff
@@ -158,6 +162,11 @@
   padding: 0.3rem 0;
 }
 
+.px7-5 {
```

Comment:
It seems like previous bootstrap (< 5) applied a left/right 7.5 padding to the `col` class. This helper is used to tilt back the `Mempool` homepage section to where it originally were.